### ISSUE TITLE
Default ardi.json includes absolute paths, which is problematic when checked in

### ISSUE THREAD

**Issue by egnor** [robgonnella/ardi#60]
`ardi add build` adds entries like this by default to `ardi.json`:

```
	"builds": {
		"mpr121_mega": {
			"directory": "/home/egnor/source/machine/arduino/TESTS/mpr121_tester",
			"sketch": "/home/egnor/source/machine/arduino/TESTS/mpr121_tester/mpr121_tester.ino",
			"baud": 115200,
			"fqbn": "arduino:avr:mega",
			"props": {}
		}
	}
```

Since an `ardi.json` file is often checked in to source control (e.g. git) and checked out elsewhere, this is problematic since the absolute paths don't work elsewhere.

For me, at least, changing "directory" to "." and "sketch" to a local path seems to work, though this may have other side effects?

**Comment by robgonnella** [comment]
good catch, will update this to use relative paths in `ardi.json` and construct absolute paths for build and upload requests

**Comment by robgonnella** [comment]
Updated in new beta version, you can try it out via `GO111MODULE=on go get github.com/robgonnella/ardi/v2@v2.3.0-beta.1`

**Comment by egnor** [comment]
Verified works!

**Comment by robgonnella** [comment]
Closing: Addressed in v2.3.0
